### ISSUE TITLE
Switch between light mode and dark mode, depending on the times

### ISSUE THREAD

**Issue by H5820121** [darkreader/darkreader#11601]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this feature request to avoid opening a duplicate.

### Is this feature request related to a problem?

No

### Feature Request Description

Can you please add an option, switch between light mode and dark mode, depending on the times. I mean when Dark Reader is active all the time, but only goes from light to dark.

This app is great!
Thanks!

**Comment by austinhyde** [comment]
Do you mean this?
<img width="270" alt="image" src="https://github.com/darkreader/darkreader/assets/694995/60e58e2e-342e-4d4a-a70d-12e08ebb052f">

(after clicking the "Configure Automation" link under the on/off toggle)

**Comment by H5820121** [comment]
>Do you mean this?

No, ""Configure Automation" just switching between ON and OFF, I mean switching between bright and dark (when it's always ON).

![ScreenShot_20230815091911](https://github.com/darkreader/darkreader/assets/105562828/0acec9cf-cec0-4917-a78a-61037001fb6c)

Thanks!

**Comment by freebrowser1** [comment]
Same here. Ubuntu 22.04 LTS, Brave [Version 1.56.14 Chromium: 115.0.5790.114 (Official Build) (64-bit)](https://brave.com/latest/)
OS is in dark mode and Brave as well, but Darkreader stays bright. And when I set it time based, e.g. dark from 21:00 then it does not automatically switch at 9pm.

**Comment by woj-tek** [comment]
> (after clicking the "Configure Automation" link under the on/off toggle)

I have the same issue with DR doesn't switch to dark mode automatically with times configured. If I open the time panel, select "start" and adjust it (even by minute, doing up/down while already in range) then it kicks in and turns into dark mode. It looks like some listener is not working or some event not firing?

**Comment by joeriexelmans** [comment]
Not a solution to what seems to be a bug here, but I set Dark Reader to follow the OS setting, which does work (on Linux).

The OS can then be configured to switch at specific times, but I prefer switching manually: a single keyboard shortcut makes my OS go dark or light, and Dark Reader follows.

**Comment by freebrowser1** [comment]
> Not a solution to what seems to be a bug here, but I set Dark Reader to follow the OS setting, which does work (on Linux).
> 
> The OS can then be configured to switch at specific times, but I prefer switching manually: a single keyboard shortcut makes my OS go dark or light, **and Dark Reader follows**.
For me not, at least not on Ubuntu 22.04 LTS (Brave browser), but on macOS 13.4 it does work, both Firefox and Brave.

**Comment by woj-tek** [comment]
> Not a solution to what seems to be a bug here, but I set Dark Reader to follow the OS setting, which does work (on Linux).

same issue as @freebrowser1 - on macOS it doesn't switch to dark mode even though everything else switches just fine. And because of that I opted for time-based switch but it's not working either...
